data/json/recipes/recipe_others.json

```diff
@@ -360,7 +360,7 @@
     "time": 5000,
     "autolearn": true,
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
-    "components": [ [ [ "stick", 1 ], [ "2x4", 1 ] ], [ [ "rock", 1 ] ], [ [ "filament", 40, "LIST" ], [ "cordage_short", 2, "LIST" ] ] ]
+    "components": [ [ [ "stick", 1 ], [ "2x4", 1 ] ], [ [ "rock", 1 ] ], [ [ "cordage_short", 2, "LIST" ], [ "filament", 40, "LIST" ] ] ]
```

Comment:
I feel that the culprit is in this line. Maybe split it into 

```
"components": [
  [ [ "stick", 1 ], [ "2x4", 1 ] ], 
  [ [ "rock", 1 ] ], 
  [ [ "cordage_short", 2, "LIST" ], [ "filament", 40, "LIST" ] ] 
]
```

this?

Reply:
If it is then http://dev.narc.ro/cataclysm/format.html needs to be updated again, because this is the way it's telling me is correct.

Evidently back before my time the error-checker was less worthless, as I've found. If it told me what the problem actually is like it used to I wouldn't be struggling to figure out what it actually wants.

Issue reported here: https://github.com/CleverRaven/Cataclysm-DDA/issues/23105